### PR TITLE
Update conferences.yml

### PR DESCRIPTION
**Pull request by yizheng-chen**
update ACM mm 2021

### CODE REVIEW

File: _data/conferences.yml [hunk 1]
```diff
@@ -305,13 +305,13 @@
...
 - title: MM
   hindex: 58
-  year: 2020
-  id: mm20
-  link: https://2020.acmmm.org/
-  deadline: '2020-05-24 23:59:00'
+  year: 2021
+  id: mm21
+  link: https://2021.acmmm.org/
+  deadline: '2020-04-3 23:59:00'
```
Comment (abhshkdz) [inline _data/conferences.yml]: Should this be 2021-04-03?

File: _data/conferences.yml [hunk 2]
```diff
@@ -305,13 +305,13 @@
...
   timezone: UTC-12
   date: October 12-16, 2020
```
Comment (abhshkdz) [inline _data/conferences.yml:313]: Please update this to 2021 dates

**Review by abhshkdz — CHANGES_REQUESTED** [review]

File: _data/conferences.yml (hunk 1 above)
Comment (yizheng-chen) [reply _data/conferences.yml]: Yes, I have updated it.

**Comment by abhshkdz** [comment]
Could you update the `date` field to 2021 please.

**Comment by abhshkdz** [comment]
Nvm done; thanks!

**Pull request merged**